pkg/networkmanager/networking.scss

```diff
@@ -111,7 +111,7 @@ th {
 }
 
 // If the button is not the first item in the DescriptionListDescription add left padding
-.network-interface-settings dd button:not(:first-child) {
+.network-interface-settings div.pf-c-description-list__text button:not(:first-child) {
```

Comment:
This is wrong. Never use div or span when selecting things in CSS. Also, usually try to not prefix with the element when selecting a class. (PF has famously changed elements several times, including recently.)